R/config.R

```diff
@@ -9,7 +9,7 @@
 
 #' @importFrom attempt attempt is_try_error
 guess_where_config <- function(
-  path = golem::pkg_path(),
+  path = get_golem_wd(),
```

Comment:
Given that these ones try to guess where the config.yaml is and that get_golem_wd() calls `get_golem_things()` which calls `get_current_config()` which calls `guess_where_config()`, we'll need to keep them as before if we want to avoid an infinite call stack :) 

```suggestion
  path = golem::pkg_path(),
```